### PR TITLE
Investigations

### PR DESCRIPTION
**Pull request by willrogers**
@aawdls @T-Nicholls further refinements from testing.

### CODE REVIEW

**Comment by aawdls** [comment]
Virtual accelerator now starts for me fine without the unit conversion error.

File: pytac/units.py [hunk 1]
```diff
@@ -178,22 +178,24 @@ def phys_to_eng(self, value):
...
-            raise UnitsException("{0}: no conversion result "
-                                 "within conversion limits ({1}, "
-                                 "{2}).".format(self,
+            valid_results = [r for r in valid_results if r <= self.upper_limit]
+        if len(valid_results) == 1:
+            return valid_results[0]
+        elif len(valid_results) == 0:
+            raise UnitsException("{0}: none of conversion results {1} "
+                                 "within conversion limits ({2}, "
+                                 "{3}).".format(self, results,
```
Comment (aawdls) [inline pytac/units.py]: do we need `valid_results` in the format as well?

Comment (willrogers) [reply pytac/units.py]: No, because we only get to this line if `len(valid_results) == 0`.

**Comment by T-Nicholls** [comment]
A note, `NullUnitConv` objects can have conversion limits, though are unlikely to, and so it is possible that a name might be useful for differentiating between unitconv's in this case.
This is entirely theoretical as `load_csv` does not check for conversion limits for `NullUnitConv`s, and I don't think it should until a practical use case arrives.

**Pull request merged**